app/serialised_models.py

```diff
@@ -78,10 +78,10 @@ class SerialisedService(SerialisedModel):
         "name",
         "active",
         "contact_link",
+        # TODO: do we still need this?
         "normalised_service_name",
-        # TODO: add these fields when data migrated. check how these interact with hybrid properties on model.
-        # "custom_email_sender_name",
-        # "email_sender_local_part",
+        "custom_email_sender_name",
+        "email_sender_local_part",
```

Comment:
I wonder if it’s worth adding these in a separate PR first and then clearing the cache